onnxruntime/test/python/transformers/bert_model_generator.py

```diff
@@ -26,13 +26,13 @@ def reverse_if(inputs, reverse=False):
         inputs.reverse()
     return inputs
 
-
 def create_bert_attention(input_hidden_size=16,
```

Comment:
nit: maybe keep the same sized qkv scenario as the default parameters, as it's the most common case at the moment

Reply:
preserved the original test. but makes sense to update to the most common case.